website/content/en/highlights/2022-12-22-enhanced-metric-tags.md

```diff
@@ -0,0 +1,46 @@
+---
+date: "2022-12-22"
+title: "Enhanced Metric Tags"
+description: "The metric data model now supports an enhanced representation of tag values"
+authors: ["bruceg"]
+pr_numbers: [14742]
+release: "0.27.0"
+hide_on_release_notes: false
+badges:
+  type: enhancement
+---
+
+The existing set of tags that have been available on metrics was a simple mapping of string values
+to single string keys. Some sources, however, may produce metrics that contain tags that don't fit
+into this model.
```

Comment:
```suggestion
The existing set of tags that have been available on metrics was a simple mapping of string values
to single string keys. Some sources, however, may produce metrics that contain tags that don't fit
into this model. They can produce tags that are static strings (rather than key/value) or that have multiple
values for a given key.
```

Reply:
Updated in 4656d788c